source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -299,15 +400,36 @@ def _get_matrix(cls, json_response, origins, destinations):
 
         return sn_routing_matrix
 
-    def get_matrix_params(self, origins, destinations, mode, max_duration, request):
+    @classmethod
+    def _create_matrix_response_with_direct_path(cls, json_response, origin, destination, row):
+        routing = row.routing_response.add()
+        routes = json_response.get('response', {}).get('route', [])
+        if not routes:
+            return row
+        route = routes[0]
+        travel_time = route.get('summary', {}).get('travelTime')
+        if travel_time:
+            routing.duration = travel_time
+            routing.routing_status = response_pb2.reached
+        else:
+            # since we limit the number of points given to HERE, we say that all the others cannot be
```

Comment:
Does this comment applies here ? There is not limit on the number of points for a direct path query right ?

Reply:
Exactly, this comment is an evil copy/paste ! 
Nice catch